starter-code/MongooseMovies/routes/celebrities.js

```diff
@@ -0,0 +1,89 @@
+const express = require('express');
+const Celebrity = require('../models/celebrity');
+const router = express.Router(); // what is Router?
+
+
+router.get('/', (req, res, next) => { // why just '/' for the URL?
+  // console.log("debug" + req.params);
+  Celebrity.find({},(err, celebrities) => {
+    if (err) {
+      return next(err);
+    }
+    res.render('celebrities/', {
+      celebrities: celebrities
+    });
+  });
+});
+
+router.get('/:id/show', (req, res, next) => {  // don't understand why it's id/show and why without /show it doesn't work
```

Comment:
You could set this url on "/:id" only, if you want, and then reroute the user to that url instead.

Reply:
You have to make sure that the browser is directed to the proper placem though, according to user behavior